storage/google/src/server/wrappers/Helpers.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+export type GoogleStorageConfigType = "read" | "write" | "user";
+
+export function roleFromConfigType(
```

Comment:
How does user and read differ?

Reply:
Ah I see, user is both read and write. I would have assumed that write  granted storage.objects.get
storage.objects.list